Harvest.Web/ClientApp/src/types.ts

```diff
@@ -181,3 +181,24 @@ export interface ProjectAccount {
   name: string;
   percentage: number;
 }
+
+export interface RequestInput {
+  id: number;
+  start: string;
+  end: string;
+  crop: string;
+  cropType: string;
+  requirements?: string;
+  principalInvestigator: PrincipalInvestigator;
+}
+
+export interface PrincipalInvestigator {
```

Comment:
@wongband this looks a lot like the `User` type from this file, which has everything except nameAndEmail which isn't really something they input or use so I don't think it's something that needs to be error checked.  so maybe you can just use that?  I think eventually we'll want to also replace RequestInput with some subclass of `Project` but that will take more work so you can leave that part as-is

Reply:
I created PrincipalInvestigator since nameAndEmail seems to be an input when the button is submitted. This is what we get when the user tries to create a request:

```json
{
"id":0,"firstName":"Scott",
"lastName":"Aldinger",
"email":"saldinger@ucdavis.edu",
"iam":"1000153077",
"kerberos":"scott",
"name":"Scott Aldinger",
"nameAndEmail":"Scott Aldinger (saldinger@ucdavis.edu)"
}
```

Reply:
IIRC It's just ignored once sent to the server so we don't need to validate.  But i guess it's not a problem since they won't have control to mess up the values anyway.  I'm fine if you leave it as is if it's not a quick change, I don't really remember how that value gets there without looking more.

Reply:
Okay, so I changed it so we don't have to create a whole new type.